src/main/java/com/team766/logging/Logger.java

```diff
@@ -10,6 +13,10 @@
 import java.util.EnumMap;
 
 public final class Logger {
+
+    private static final boolean ALSO_LOG_TO_DATALOG =
+            (RobotProvider.instance instanceof WPIRobotProvider);
```

Comment:
This feels like it's at risk for initialization order bugs; i.e. if the Logger is called while Maroon Framework is still initializing.

Reply:
One possible approach could be to invert the dependency here: set `Logger.ALSO_LOG_TO_DATALOG = true;` in the constructor for `WPIRobotProvider`

Reply:
or maybe `com.team766.hal.wpilib.RobotMain` would be a better place